### PR TITLE
[IMP] Link trackers: updating content

### PR DESCRIPTION
**Pull request by Donapi**
[taskid-3754778](https://www.odoo.com/web#id=3754778&cids=1&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/websites/website/reporting/link_tracker.rst [hunk 1]
```diff
@@ -2,55 +2,78 @@
...
-Set up traceable URLs
----------------------
+For each attribute available in the :guilabel:`Visibility` section, you can choose
+:guilabel:`Visible for` or :guilabel:`Hidden for` and choose the record you want from the dropdown
+list.
 
-Go to :menuselection:`Website --> Go to website --> Promote --> Track this page`. Here, you are able
-to get a specific tracked URL based on the campaign, medium, and source being used.
+.. seealso::
+   :doc:`Building blocks <../web_design/building_blocks>`
```
Comment (masp-odoo) [inline content/applications/websites/website/reporting/link_tracker.rst:53]: Hello, je mettrais peut-être juste le lien avec l'ancre qui emmène directement à la visibility des building blocks :)
https://www.odoo.com/documentation/17.0/applications/websites/website/web_design/building_blocks.html#visibility

**Review by masp-odoo — COMMENTED** [review]
Hello @Donapi, une toute petite suggestion :)
Beau travail sinon !

File: content/applications/websites/website/reporting/link_tracker.rst (hunk 1 above)
Comment (Donapi) [reply content/applications/websites/website/reporting/link_tracker.rst:53]: J'ai mis le lien sur "visibilty" dans le paragraphe juste avant, bonne suggestion :)

File: content/applications/websites/website/reporting/link_tracker.rst [hunk 2]
```diff
@@ -2,55 +2,75 @@
...
+driving traffic and conversions.
 
-Go to :menuselection:`Website --> Configuration --> Settings` and activate *Link Trackers*.
+Website visibility
+------------------
 
-.. image:: link_tracker/enable_link_tracker.png
-   :align: center
-   :alt: View of Website settings page emphasizing the link trackers field in Odoo Website
+The :abbr:`UTM (Urchin Tracking Module)` parameters allow you to display some site elements only for
```
Comment (xpl-odoo) [inline content/applications/websites/website/reporting/link_tracker.rst:40]: I wouldn't say it's the UTM parameters that _allow you_ to do something.
I would say that you can hide/show elements according to the UTM parameters.

File: content/applications/websites/website/reporting/link_tracker.rst [hunk 3]
```diff
@@ -2,55 +2,75 @@
...
-Go to :menuselection:`Website --> Go to website --> Promote --> Track this page`. Here, you are able
-to get a specific tracked URL based on the campaign, medium, and source being used.
+For each attribute available in the `Visibility <https://www.odoo.com/documentation/17.0/applications/websites/website/web_design/building_blocks.html#visibility>`_ section, you can choose
+:guilabel:`Visible for` or :guilabel:`Hidden for` and choose the record you want from the dropdown
+list.
 
-.. image:: link_tracker/link_tracker_fields.png
-   :align: center
-   :alt: View of the link tracker fields for Odoo Website
+Your tracked links
```
Comment (xpl-odoo) [inline content/applications/websites/website/reporting/link_tracker.rst:52]:
```suggestion
Tracked links overview
```

File: content/applications/websites/website/reporting/link_tracker.rst [hunk 4]
```diff
@@ -2,55 +2,75 @@
...
+Statistics
+----------
 
-To look at statistics of your links, go to :menuselection:`Website --> Go to website --> Promote
---> Track this page`. Besides being able to see the *Most Clicked* and *Recently Used* links, you
-can also see complete statistics by clicking on *Stats*, including the number of clicks, and the
-country of origin for those clicks.
+Using tracked links provides the advantage of obtaining statistics to assess the performance of your
+campaign, medium, and source. This allows you to determine the number of customers who visited your
+website through the tracked link.
```
Comment (xpl-odoo) [inline content/applications/websites/website/reporting/link_tracker.rst:66]:
```suggestion
```
Already mentioned in the intro, maybe just add "To measure the performance of the tracked links,..."

**Review by xpl-odoo — CHANGES_REQUESTED** [review]

File: content/applications/websites/website/reporting/link_tracker.rst [hunk 5]
```diff
@@ -2,55 +2,75 @@
 Link trackers
 =============
 
-Link Trackers allow you to track your marketing campaigns (emails, banner ads, blog posts, social
-media posts, affiliate links, etc.). This way, you are able to identify your best traffic sources
-and make informed decisions about the distribution of your marketing budget.
+Link trackers allow you to trace the origins of your website traffic, whether from emails, banner
+ads, blog posts, social media posts, affiliate links, or other sources. They let you determine which
+channels bring you the most visitors and make informed decisions.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/reporting/link_tracker.rst:7]: updated

File: content/applications/websites/website/reporting/link_tracker.rst [hunk 6]
```diff
@@ -2,55 +2,75 @@
...
 
-Configuration
-=============
+Create a traceable URL
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/reporting/link_tracker.rst:9]: updated

File: content/applications/websites/website/reporting/link_tracker.rst [hunk 7]
```diff
@@ -2,55 +2,75 @@
...
+#. :guilabel:`URL`: This field contains the URL of the page you want to track, such as your home
+   page or one of your product’s pages. It is automatically populated with the URL page from which
+   you access the menu. I.e., if you go to the link tracker page from one of your product pages, the
+   URL will be the one of that page.
+
+#. :guilabel:`Campaign`: The specific campaign the link is associated with. For example, if you run
+   multiple campaigns on Facebook simultaneously, you can differentiate between them using this
+   parameter.
+
+#. :guilabel:`Medium`: The channel to share your link, such as an email or a social media ad.
```
[the remark this answers is not in the document]
Comment (Donapi) [reply content/applications/websites/website/reporting/link_tracker.rst:25]: updated

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**